plugins/MobileMessaging/SMSProvider/FreeMobile.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\MobileMessaging\SMSProvider;
+
+use Piwik\Http;
+use Piwik\Notification;
+use Piwik\Plugins\MobileMessaging\APIException;
+use Piwik\Plugins\MobileMessaging\SMSProvider;
+
+/**
+ * Used for testing
+ *
+ */
+class FreeMobile extends SMSProvider
+{
+
+    const API_URL = 'https://smsapi.free-mobile.fr/sendmsg';
+    const SOCKET_TIMEOUT = 15;
+
+    public function verifyCredential($apiKey)
+    {
+        $account = explode(" ", $apiKey);
+        if (2 != count($account)) {
+            throw new APIException(
+                'API key must to contain the user and password separate by space.'
```

Comment:
I think it should be `API key must to contain the username and password separated by space.`